uui/components/fileUpload/DropSpot.tsx

```diff
@@ -9,10 +9,25 @@ import { i18n } from '../../i18n';
 import css from './DropSpot.module.scss';
 
 export interface DropSpotProps extends IHasRawProps<React.HTMLAttributes<HTMLDivElement>> {
+    /*
+    * CSS class(es) to put on component's root. See {@link https://github.com/JedWatson/classnames#usage} for details
+    */
     cx?: string;
+    /*
+    * Defines the infoText.
```

Comment:
Add more clear description what is the infoText - where it will be rendered and for which cases?

Reply:
fixed